locales/en.js

```diff
@@ -72,5 +72,12 @@
 	"ValidReason": "Please provide a reason (more than 20 char)",
 	"retrievedSuccessfully": "Requests retrieved successfully",
 	"InvalidParams": "Invalid params",
-	"managerEmailNotExist": "Manager is not found, choose another one"
-}
+	"managerEmailNotExist": "Manager is not found, choose another one",
+	"notifyInApp is required": "notifyInApp is required",
+	"notifyByEmail is required" : "notifyByEmail is required",
+	"notifyInApp can either be true or false": "notifyInApp can either be true or false",
+	"changedpreferences" : "Successfully changed preferences",
+	"noNotification" : "You have no notifications",
+	"noNewNotification" : "You have no new notifications",
+	"notManager" : "To do this, You have to be a manager"
+}
```

Comment:
```suggestion
}

```
Consider adding a new line to the end